docs/src/pages/customization/default-theme/DefaultTheme.js

```diff
@@ -64,14 +64,45 @@ function useTokenType(type) {
   }
 }
 
+const useObjectEntryLabelStyles = makeStyles({
+  colorPreview: {
+    margin: '0 5px 0 5px',
+  },
+});
+
 function ObjectEntryLabel({ objectKey, objectValue }) {
   const type = useType(objectValue);
   const label = useLabel(objectValue, type);
   const tokenType = useTokenType(type);
+  const classes = useObjectEntryLabelStyles();
+
+  const includeColorSample =
```

Comment:
We should use the raw type instead since we might change what kind of node we return from `useLabel`. And then it should be possible to look at the first token that indicates the color i.e. `/^(#|rgb|rgba|hsl|hsla)/`.

So far I don't understand what each branch checks for.

Also: We can move this into `useLabel` and return the proper node from there.

Reply:
I agree, moving into `useLabel` would be cleaner, I can move it to there.

The only thing with this logic - `/^(#|rgb|rgba|hsl|hsla)` is it picks up rgbas with just alphas.   i.e. `divider:  rgba(0,0,0,0.12)` would look differently depending if it was on a black or white (or some other color) background so I was going to skip showing that value.

Reply:
I would rather display colors everywhere than sometimes not.  Colors look different depending on the surrounding anyway (especially tiny squares).